core/include/irods/private/http_api/openid.hpp

```diff
@@ -14,12 +14,34 @@
 
 namespace irods::http::openid
 {
+	enum class TokenType
+	{
+		Access,
+		ID
+	};
+
 	auto create_oidc_request(boost::urls::url_view _url)
 		-> boost::beast::http::request<boost::beast::http::string_body>;
 
+	/// Validates an OAuth 2.0 Access Token using the Introspection Endpoint.
+	/// See RFC 7662 on OAuth 2.0 Token Introspection for more details.
+	///
```

Comment:
Can we document the input param while we're in here?

Reply:
Done.